### PR TITLE
Tag color variables

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Currently, primitive colors are used in the proposal status tags. To make it more organized, we are creating separate variables for each proposal status (except for the status Unknown, as it’s not a real status and uses a ~~disabled~~ default style).

# Changes

- Added new variables that match the design.

# Screenshots

No visual changes yet.

### CODE REVIEW

File: src/lib/styles/themes/dark.scss [hunk 1]
```diff
@@ -218,4 +218,15 @@
   --negative-emphasis: var(--pink-200);
   --negative-emphasis-contrast: var(--neutral-50);
   --negative-emphasis-light: var(--pink-200-a25);
+
+  --proposal-open-background: var(--blue-accent-tint);
```
Comment (yhabib) [inline src/lib/styles/themes/dark.scss:222]: do I understand correctly that our pattern is something like `component-state-property`?

Comment (mstrasinskis) [reply src/lib/styles/themes/dark.scss:222]: Good point! I found them in the new design variables in Figma. I’ll rename them accordingly.
image

File: src/lib/styles/themes/dark.scss [hunk 2]
```diff
@@ -218,4 +218,18 @@
   --negative-emphasis: var(--pink-200);
   --negative-emphasis-contrast: var(--neutral-50);
   --negative-emphasis-light: var(--pink-200-a25);
+
+  --elements-tag-background: var(--night-575);
```
Comment (dskloetd) [inline src/lib/styles/themes/dark.scss:222]: I'm wondering if we really need the "elements" prefix here. Do you know what purpose it serves in Figma?

Comment (mstrasinskis) [reply src/lib/styles/themes/dark.scss]: Good point; let’s keep it simple and flatter. After syncing with Artem, we decided to remove the elements. Updated.

**Review by dskloetd — APPROVED** [review]

**Pull request merged**